pkgs/tools/networking/eternal-terminal/default.nix

```diff
@@ -42,7 +42,7 @@ stdenv.mkDerivation rec {
     "-std=c++17"
   ];
 
-  LDFLAGS = lib.optional stdenv.cc.isClang [
+  LDFLAGS = lib.optional (stdenv.cc.isClang && !stdenv.isDarwin) [
     "-lc++fs"
   ];
 
```

Comment:
Doesn't seem darwin-specific. Likely not needed after clang was bumped to version 11 on all platforms.
```suggestion
```

Reply:
@veprbl thanks for the info. I do have a linux machine that I can test this on as well. I'll report back shortly...

Reply:
omitting this flag on linux seems to work as well.